libobs/obs-output.c

```diff
@@ -1592,7 +1721,7 @@ static bool initialize_interleaved_packets(struct obs_output *output)
 
 	/* ensure that there is audio past the first video packet */
```

Comment:
This should probably detect the first video packet on *any* video encoder, to keep with the original intention of only sending audio after video has started.

Additionally, the current implementation assumes that there is a video encoder at idx 0, which it may be possible that there isn't one? This bug would also apply to the multiple audio track implementation if true.